src/applicants/applicantsRoutes.js

```diff
@@ -6,6 +6,7 @@ import applicantsController from "./applicantsController";
 const router = express.Router();
 
 router.get("/all/:page", isAuth, applicantsController.getAllApplicants);
+router.get("/:id", applicantsController.getApplicant);
```

Comment:
Should this route be protected as well, so that only logged in user can access it?

Reply:
Added isAuth middleware